### PR TITLE
Add overview to README

### PR DESCRIPTION
**Pull request by bitdivision**
This repo is currently missing a general overview for what the tool does. The PR adds a rough overview of the functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # CP-8 Cli
```
Comment (hahmed) [inline README.md:1]: Would you mind updating this to CP8 CLI?

Comment (hahmed) [inline README.md]: Would you mind updating this to CP8 CLI?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+CP-8 is a ruby tool which aids in interacting with github by opening draft PRs, creating suggestion branches, opening CI for a branch etc.
```
Comment (hahmed) [inline README.md]: What do you think about this?

```
CP8 is a command line tool which aids in interacting with GitHub.

CP8 helps you create draft pull requests, suggests branch names and opens your CI for a branch.
```

**Review by hahmed — COMMENTED** [review]
Thank you so much for your contribution @bitdivision I had a few suggestions, WDYT?

File: README.md (hunk 2 above)
Comment (bitdivision) [reply README.md]: Thanks, this looks great. I've updated the PR.

**Pull request merged**